trussworks/react-uswds
# feat: Update to USWDS 2.9.0

# Summary

Bumps USWDS dependency. Additionally makes react-uswds changes to mimic [2.9.0 bugfixes and improvements](https://github.com/uswds/uswds/releases/tag/v2.9.0) 

## Related Issues or PRs

closes: https://github.com/trussworks/react-uswds/issues/1046

## How To Test

Storybook and unit tests are the best bet for testing this.

Specifically, take note of the warm and cool button accent stories

Additionally, changes to Legend styles can be seen in the Fieldset stories. Old legend styles are preserved with the legendStyle="large" prop. (https://github.com/uswds/uswds/pull/3465). See note below regarding Happo

### Other
- Note: Several negligible Happo diffs due to font smoothing on buttons ,etc with 2.9.0 will be expected. One that is potentially more concerning is due to the changes to legend classes. By default, legends now render with smaller styles as the "old"/large legends are not recommended by uswds:
![image](https://user-images.githubusercontent.com/15805554/112651530-de078880-8e22-11eb-8ece-dd37cede3b16.png)
Not also that this now matches the USWDS component preview: 
![image](https://user-images.githubusercontent.com/15805554/112651695-05f6ec00-8e23-11eb-95c1-b2a1f28fe1e8.png)


- Should these changes be approved, I will add issues for removing deprecated Button and Fieldset props in the V2 milestone.
